repl/repl_test.go

```diff
@@ -2570,10 +2570,17 @@ query:1     | Redo data.a[i].b.c[j] = x
 +---+---+---+---+
 | 0 | 1 | 1 | 2 |
 +---+---+---+---+`)
-	expected += "\n"
 
-	if expected != buffer.String() {
-		t.Fatalf("Expected output to be exactly:\n%v\n\nGot:\n\n%v\n", expected, buffer.String())
+	actual := ""
+	for _, line := range strings.Split(buffer.String(), "\n") {
+		if !strings.Contains(line, "| Unify") {
```

Comment:
So we're filtering out the Unify events -- wouldn't it make sense to adjust the expectation instead? 🤔

Reply:
@srenatus I was a bit worried about the ordering of these, but seems like its fully deterministic and its easy enough to compare and paste in the expected output when it changes.